utils/zerovec/src/zerovec/slice.rs

```diff
@@ -237,25 +237,6 @@ where
     /// if and only if `T::ULE` and `P::ULE` are the same size.
     ///
     /// If `T` and `P` have the exact same `ULE`, use [`Self::cast()`].
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use zerovec::ZeroSlice;
-    ///
-    /// const bytes: &[u8] = &[0x7F, 0xF3, 0x01, 0x00, 0x49, 0xF6, 0x01, 0x00];
-    /// const zs_u32: &ZeroSlice<u32> = {
-    ///     match ZeroSlice::<u32>::try_from_bytes(bytes) {
```

Comment:
Suggestion: as above, use `ZeroSlice::<[u8; 3]>` as the new reference type.

Deleting this docs test means we reduce test coverage, which isn't great.

Reply:
Same comment as above, `[u8; 3]` is still not equivalent to `char` which is of 4 bytes

Reply:
Added a u32<->i32 test